chef/data_bags/crowbar/template-network.json

```diff
@@ -257,7 +258,7 @@
     "network": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 20,
+      "schema-revision": 100,
```

Comment:
I guess I should have been careful about asking for cherry-pick: we want 21 here. Apologies, my bad.